### DISCUSSION TITLE
Starting deleted beatmap throws service message user shouldn't receive

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#27716] [Q&A] [answered]
Beatmapset has been deleted and overriden https://osu.ppy.sh/beatmapsets/2084309#osu/4365910

https://github.com/ppy/osu/assets/86934170/c10e6c65-34dd-4b08-996a-059dc92e2ae8

**Comment by Liam-DeVoe** [comment]
Could you attach your logs?

#### Desktop platforms

- Open the game settings and click on "Export logs"
- Click the notification to locate the file
- Drag the generated .zip files into the github issue window

#### Mobile platforms

- On Android, navigate to Android/data/sh.ppy.osulazer/files/logs using a file browser app.
- On iOS, connect your device to a PC and copy the logs directory from the app's document storage using iTunes. (https://support.apple.com/en-us/HT201301#copy-to-computer)

**Reply by Loreos7** [reply]
[compressed-logs.zip](https://github.com/ppy/osu/files/14736780/compressed-logs.zip)

**Reply by Liam-DeVoe** [reply, marked as answer]
thanks, I've promoted to an issue: https://github.com/ppy/osu/issues/27720
